ibridgesgui/login.py

```diff
@@ -86,6 +95,9 @@ def login_function(self):
         try:
             if self.cached_pw is True and self.password_field.text() == "***********":
                 self.logger.debug("Login with %s and cached password.", env_file)
+                with open(IRODSA, "w",  encoding="utf-8") as f:
+                    f.write(self.prev_settings[str(env_file)])
```

Comment:
I don't know exactly what the env_file object is, but perhaps it would be useful to use the full path? It might not be an issue now, but if something somehow changes the cwd that might be a very difficult issue to track down.

Reply:
The env_file is indeed the full path to the environment.json:

```
env_file = self.irods_config_dir.joinpath(self.envbox.currentText())
```

And in the ibridges-gui.json we store the full path.